config.toml

```diff
@@ -1,4 +1,4 @@
-testing = false
+testing = true
```

Comment:
i keep seeing this flicking back and forth between different branches. which one is it supposed to be ? ?

Reply:
bad habits sorry